pyEpiabm/pyEpiabm/tests/test_utility/test_covidsim_kernel.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import numpy as np
+from parameterized import parameterized
+
+from pyEpiabm.utility.covidsim_kernel import SpatialKernel
+
+numReps = 5
+
+
+class TestSpatialKernel(unittest.TestCase):
+    """Test the 'SpatialKernel' class.
+    """
+    @parameterized.expand([(np.random.rand(1) * numReps,)
+                          for _ in range(numReps)])
+    def test_kernel(self, dist):
+        dist = 10 * dist[0]
+        dist_object = SpatialKernel()
+        value = dist_object.weighting(dist)
+        self.assertTrue(0 <= value)
+
+    @parameterized.expand([(np.random.rand(3) * numReps,)
+                           for _ in range(numReps)])
+    def test_kernel_shape_size(self, dist):
+        dist = 10 * dist
+        dist_object = SpatialKernel()
```

Comment:
Do we need to create an instance of the class if we are only calling a static method?

Reply:
nope